model/src/main/proto/javaclasses/mealorder/c/commands.proto

```diff
@@ -159,6 +159,7 @@ message CancelOrder {
     OrderId order_id = 1;
 
```

Comment:
// An identifier of the menu which contains dishes that can be added to an order. (above)

Reply:
// Adds a dish to an order.